script.js

```diff
@@ -20,76 +20,35 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
   back.classList.remove('active');
 }));
 
-// form validation
-const form = document.querySelector('form');// get the form
-const username = document.getElementById('name');// get the name
-const email = document.getElementById('email');// get the email
-const message = document.getElementById('message');// get the message
-const errorDisplay = document.querySelector('.errorMsg');
-// for setting the error message
-const setError = (element, message) => {
-  errorDisplay.innerText = message;
-  errorDisplay.classList.add('error');
-  errorDisplay.classList.remove('success');
-};
-
-// for setting the the success message
-const setSuccess = () => {
-  errorDisplay.innerText = '';
-  errorDisplay.classList.add('success');
-  errorDisplay.classList.remove('error');
-};
-
-let submitPt = 0;// for calcuating the result should be 3
-
-const validateInputs = () => {
-  const usernameValue = username.value.trim();
-  const emailValue = email.value.trim();
-  const messageValue = message.value.trim();
-
-  // for name validation
-  if (usernameValue === '') {
-    setError(username, 'Username is required');
-    return;
-  } if (usernameValue.split('').length > 30) {
-    setError(username, 'Username should not exceed 30 characters');
-    return;
-  }
-  setSuccess(username);
-  submitPt += 1;
-
-  // for email validation
-  if (emailValue === '') {
-    setError(email, 'Email is required');
-    return;
-  } if (emailValue !== emailValue.toLowerCase()) {
-    setError(email, 'Email should be in lowercase');
-    return;
-  }
-  setSuccess(email);
-  submitPt += 1;
-
-  // for message validation
-  if (messageValue === '') {
-    setError(message, 'Please add a message');
-  } else if (messageValue.split('').length > 500) {
-    setError(message, 'Message should be short and do not exceed 500 characters');
+const formbutton = document.getElementById('bot');
+const name = document.querySelector('#name');
+const message = document.querySelector('#message');
+const form = document.querySelector('#form-action');
+const Email = document.getElementById('email');
+const Errormessage = document.getElementById('error-message');
+Errormessage.textContent = 'Message not sent! Please write using only the small letters. Thank you.';
+
+function validationForm(e) {
+  if (Email.value !== Email.value.toLowerCase()) {
+    Errormessage.style.display = 'block';
+    e.preventDefault();
   } else {
-    setSuccess(message);
-    submitPt += 1;
+    Errormessage.style.display = 'none';
   }
-};
-
-// the validation when the form is submitted
-form.addEventListener('submit', (e) => {
-  e.preventDefault();// prevent sumitting
-
-  validateInputs();// run checks
-
-  // deciding to submit or not
-  if (submitPt === 3) {
-    form.submit();
-  } else {
-    submitPt = 0;
-  }
-});
\ No newline at end of file
+}
+formbutton.addEventListener('click', validationForm);
+
+form.addEventListener('submit', () => {
```

Comment:
- [ ] You save the data to the browser after submitting the form. You should preserve data in the browser when filing the form and not submitting it, the entered values should be saved in the browser, and when the user changes the content of any input field in the form the data object is updated and it is saved to the local storage.

Reply:
@Ismailco thank you for the feedback so what should I change and put in my code because I have no idea what to do next

Reply:
@cynthia-nwannah, Sorry I was away from my computer. I was gonna suggest using `input` instead of `submit` but I see that you fixed the issue and got approved, 👍🏼 Great work.